### ISSUE TITLE
Jellyfin for WebOS 4

### ISSUE THREAD

**Issue by dustingrady** [jellyfin/jellyfin-webos#138]
Recently WebOS 5 support was added (whoo!) and I was one of those to find that they are actually running WebOS 4.
I read that WebOS 3 and 4 are both only awaiting the resolution of issue #**9430** (can't find it in the main repo), which is a playback bug.

Are there any updates on **9430**, and in turn WebOS 3/4 apps being submitted to LG? :)

**Comment by anthonylavado** [comment]
It's actually more than that now, LG raised some new bugs to us in the webOS 5 testing, and they require them to be fixed before we submit again (even for older versions).

**Comment by lexcorp** [comment]
Hi @anthonylavado any update about this for version webos 4? I was one of those to find that they are actually running WebOS 4.  I am looking forward to this version as I also intend to integrate my jellyfin server with Home Assistant to run some automations. 

Thanks for your job...

**Comment by lexcorp** [comment]
Hola @anthonylavado lamento la insistencia, pero quisiera saber si tienes proyectada alguna liberación para la versión 4 de webos. Si hay algo en lo que pueda ayudar…

Saludos…

**Comment by TimberAngry** [comment]
Yes please 👍 C9 OLED owners like myself would love this.

**Comment by lexcorp** [comment]
Hi, like every month I'm here to find out if there's any progress for version 4 :(

**Comment by Zippy1970** [comment]
I'm guessing Jellyfin for WebOS is now a **dead** project since the last comment from the developer was from the beginning of January. Don't hold your breath waiting for a version that will work on WebOS <5

**Comment by tyhawk** [comment]
... or the developers are too busy with developing to give an update as regularly as you would like.

Please remember this is not paid software and these people are volunteering their spare time to work on this. That often means being patient and perhaps not always agreeing with the priorities of a team within their development roadmap.

**Comment by lexcorp** [comment]
I think that the development team is giving priority to the projects that have less progress or, they are waiting for the latest stable version of the Jellyfin server to be released to see how the webos client behaves.

**Comment by anthonylavado** [comment]
It is very closely tied to changes that have to happen in jellyfin-web, and 10.8 had such a drawn out cycle that it's hard to bring much else "back" into a prior version (even for a bug fix).

We also had quite a big stretch of security things we had to address, so there's finally some time/effort available to get behind this again.

We have a bespoke demo environment for validation testing by vendors (Apple, Google, Amazon, Roku, LG, and Samsung). I'm considering splitting up the LG and Samsung ones to use a custom setup that has some hot fixes to allow testing to pass. Users probably won't get the fixes until 10.9, but it's nothing groundbreaking for the moment. They are very strict and will only let some "issues" through for so long, and some are even blocking, even if they're not strictly serious issues. An example from Amazon's testing for the Android TV app - they clicked the favourite button on a demo song, and the display didn't refresh to show it as selected unless you backed out and in of the item. Easy enough visual bug to fix, but they considered it a fail, and their system automatically delists the app until you submit a working build (and also wait another week for them to test). LG will at least let you keep the previous build up.

Anyway, ranting over.

tl;dr - was drained, other issues, dependency on Web for a lot of fixes. Expect list soon. Probably custom demo for LG to accept it.

**Comment by anthonylavado** [comment]
Alright, this is how I figure this will go.

When we submitted for webOS 5, they came back with two issues that were considered "conditional pass". This means they have to be fixed for the next submission.

Current plan:

1. Create new issue pinned at top of repository, for new release actions.
2. List the two conditional pass issues, and create corresponding issues in other projects (Jellyfin-web) if necessary.
3. Gather any easy/quick fixes for now, and merge in.
4. Submit with request to deploy on all possible platforms.

They've shown that they'll update the ones where it works, and if you fail a platform, they'll skip it. With the conditional pass issues fixed, we should be easily updated for webOS 5+. For webOS 4 and below, if we pass, good. If not, then we'll have a targeted list of things to fix, and can submit again once those are fixed.

New post incoming.

**Comment by TimberAngry** [comment]
Thanks Anthony, appreciate the update :)

**Comment by anthonylavado** [comment]
Please see #169 for updates. Please do not post unless providing more information - you notify a lot of people who are following progress. Reactions are safe, they don't send notifications.  Closing this issue.
